### PR TITLE
panel meeting search route TM-3129

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do:

- [x] Add Panel Meetings to AO Menu
- [x] Add Panel Meetings to CDO Menu
- [x] Create initial landing page
- [x] Create feature flag

### CODE REVIEW

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+const PanelMeetingSearch = ({ isCDO }) => {
+  const text = isCDO ? 'yes' : 'no';
+  return (
+    <h1>
```
Comment (elizabeth-jimenez) [inline src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:6]: You wrapped everything in an <h1> 😅

Comment (elizabeth-jimenez) [reply src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:6]: I gotta go

Comment (SidelineCory24) [reply src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:6]: done

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -326,6 +336,15 @@ export const GET_PROFILE_MENU = () => MenuConfig([
             'cdo',
           ],
         } : null,
+      checkFlag('flags.panel_search') ?
+        {
+          text: 'Panel Meetings',
+          route: '/profile/cdo/panelmeetings',
+          icon: 'comment',
+          roles: [
+            'cdo',
```
[the remark this answers is not in the document]
Comment (SidelineCory24) [reply src/Constants/Menu.js:346]: done

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx]: done done

File: src/Components/BureauPage/BureauPage.jsx [hunk 3]
```diff
@@ -27,6 +28,7 @@ const BureauPage = () => {
         <Route path="/profile/ao/employeeagendas" render={() => <EmployeeAgendaSearch isCDO={false} />} />
         <Route path="/profile/ao/agendaitemhistory/:id" render={() => <AgendaItemHistory isCDO={false} />} />
         <Route path="/profile/ao/createagendaitem/:id" render={() => <AgendaItemMaintenanceContainer isCDO={false} />} />
+        <Route path="/profile/ao/panelmeetings" render={() => <PanelMeetingSearch isCDO={false} />} />
         <Route path="/profile/ao/availablebidders" render={() => <AvailableBidderContainer isCDO={false} isAO />} />
         <Route path="/profile/(bureau|ao)/dashboard" render={() => <Dashboard {...dashboardProps} />} />
         <Route path="/profile/cdo/availablebidders" render={() => <AvailableBidderContainer isCDO isAO={false} />} />
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/BureauPage.jsx:34]: bruh 🤔  (i know this isn't you - just funny)

Comment (SidelineCory24) [reply src/Components/BureauPage/BureauPage.jsx:34]: I had the same look on my face when I setup the Agenda Search routes lol

**Review by elizabeth-jimenez — APPROVED** [review]
nice!

**Pull request merged**